src/ensembl/src/shared/components/tabs/Tabs.scss

```diff
@@ -2,7 +2,7 @@
 
 .tabsContainer {
   display: flex;
-  font-size: 14px;
+  font-size: 13px;
```

Comment:
If `$global-font-size` has been changed to 13px, this class should just inherit this font-size from the body making this rule unnecessary.

Reply:
global-font-size describes the body font size (see _settings.scss and _normalize.scss). Tabs container will have inherited body font size; so specifying font-size here is redundant. In other places, such as sidebar, we may have overridden the font size with something different.

Reply:
I just noticed that in the google doc it does say body font size to be 13px so updating global-font-size is the correct way and then removing font-size in the class above. That will mean by default everywhere the font-size will be 13px unless overwritten.

Reply:
Yes thats right @ens-ma7 Its the global font size that needed change. Yes I will remove this redundant one.